app/store/index.ts

```diff
@@ -104,7 +104,7 @@ export function createStore(): Promise<Store<any>> {
     modules: {
       ...statefulServiceModules,
     },
-    strict: debug,
+    strict: false,
```

Comment:
i'm guessing this should be changed back in the final PR

Reply:
Yup, good catch!  `debug` mode is so insanely slow, I turn it off when I'm working on code that does a lot of mutations.